src/api/assets/assets.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Agreement } from './entity/agreement.entity';
+import { Question } from './entity/question.entity';
+import { Questions, AgreementData } from './data';
+import { BadRequestException } from '@nestjs/common/exceptions';
+
+@Injectable()
+export class AssetsService {
+  constructor(
+    @InjectRepository(Agreement)
+    private readonly agreementRepository: Repository<Agreement>,
+    @InjectRepository(Question)
+    private readonly questionRepository: Repository<Question>,
+  ) {}
+
+  async getQuestions(part: string) {
+    if (!['web', 'server', 'design'].includes(part)) {
+      throw new BadRequestException({
+        errors: {
+          title: 'INVALID_PART_NAME',
+          detail: '유효하지 않은 파트입니다.',
+        },
+      });
+    }
+
+    const commonQuestions = await this.questionRepository.find({
+      where: { part: 'common' },
+    });
+    const questionArray = await this.questionRepository.find({
+      where: { part },
+    });
+
+    const resultQuestions = commonQuestions.concat(questionArray);
```

Comment:
요건 concat 보다는 아래처럼 더 많이 씁니다!

```suggestion
    const resultQuestions = { ...commonQuestions, ... questionArray };
```